Tests/scripts/utils/content_packs_util.py

```diff
@@ -35,10 +35,28 @@ def is_pack_xsoar_supported(pack_path: str) -> bool:
     return pack_metadata.get(PACK_METADATA_SUPPORT, '').lower() == "xsoar"
 
 
+def is_pack_deprecated(pack_path: str) -> bool:
+    """Checks whether the pack is deprecated.
+    Tests are not being collected for deprecated packs and the pack is not installed in the build process.
+
+    Args:
+        pack_path (str): The pack path
+
+    Returns:
+        True if the pack is deprecated, False otherwise
+    """
+    pack_metadata_path = os.path.join(pack_path, PACKS_PACK_META_FILE_NAME)
+    if not os.path.isfile(pack_metadata_path):
+        return True
```

Comment:
@yuvalbenshalom @ShahafBenYakir please let me know what you think about this. It checks whether the given path to the pack_metadata file is valid (similar to the check in the function `is_pack_xsoar_supported` above), and in case it isn't it returns True which will cause the pack NOT to be collected. Is this the behavior we want or maybe the opposite (collect the pack to the installed packs)?